### DISCUSSION TITLE
Syntax highlighting in fenced code blocks

### DISCUSSION THREAD

**Discussion by GiFuJo** [enchant97/note-mark#211] [Ideas]
Hi there! I wondered if syntax highlighting in fenced code blocks would fit the projects scope. I'm using Note Mark among other things to collect some code snippets I frequently use and it would be great to have syntax highlighting for those.

**Comment by enchant97** [comment]
Good idea! Should be quick to add using [highlight.js](https://highlightjs.org/).
